### PR TITLE
APIO-230 navigate to collection with server url and type of the collection

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

File: src/consumer/apio-consumer.js [hunk 1]
```diff
@@ -9,35 +10,33 @@ import {ConversionHandler, formConverter} from '../converters';
...
 	 * @review
 	 */
-	constructor(authorizationHeaders) {
+	constructor(config = {}) {
 		this.client = new HttpClient();
 		this.parser = new JsonLDParser();
 		this.thingsCache = new Map();
-		this.conversionHandler = new ConversionHandler();
-		this.authorizationHeaders = authorizationHeaders;
+		this.conversionHandler = new ConversionHandler(this);
```
Comment (javiergamarra) [inline src/consumer/apio-consumer.js:20]: I don't like to use this around, leaks :P

File: src/navigator/navigator.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+		this.navigationSteps.push(navigationStep);
+
+		return this;
+	}
+
+	getEntryPointUrl() {
+		const {baseUrl} = this.consumer.config;
+
+		return baseUrl + '/o/api';
```
Comment (javiergamarra) [inline src/navigator/navigator.js]: Why do you need the destructuring?

File: src/navigator/navigator.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+		return this;
+	}
+
+	get() {
+		return this.executeSteps(this.navigationSteps);
+	}
+
+	executeSteps(steps) {
+		return steps.reduce(async (acc, step) => step(await acc), null);
```
Comment (javiergamarra) [inline src/navigator/navigator.js]: hehe, clever

**Review by javiergamarra — APPROVED** [review]

File: src/navigator/navigator.js (hunk 3 above)
Comment (victorg1991) [reply src/navigator/navigator.js]: @p2kmgcl helped me with this :D

File: src/navigator/navigator.js (hunk 2 above)
Comment (victorg1991) [reply src/navigator/navigator.js]: Because... destructuring all the things!

Fixed in 98d1fb5

File: src/navigator/navigator.js (hunk 3 above)
Comment (p2kmgcl) [reply src/navigator/navigator.js]: :sunglasses:

**Pull request merged**